packages/apps/plugins/plugin-debug/src/DebugPlugin.tsx

```diff
@@ -199,48 +202,62 @@ export const DebugPlugin = (): PluginDefinition<DebugPluginProvides> => {
       },
       surface: {
         component: ({ data, role }) => {
-          const { active } = data;
+          const { active, object } = data;
+
           switch (role) {
             case 'settings':
               return data.plugin === meta.id ? <DebugSettings settings={settings.values} /> : null;
             case 'status':
               return <DebugStatus />;
           }
 
-          if (!settings.values.debug) {
```

Comment:
`settings.values.debug` and `settings.values.devtools` can be enabled separately.